### DISCUSSION TITLE
Arcade/Mame-2003 high scores not saving

### DISCUSSION THREAD

**Discussion by papalozarou** [OnionUI/Onion#1192] [Q&A] [answered]
Have installed the latest OnionOS 4.2 beta on my Miyoo Mini but am having trouble with Arcade/Mame high scores not being saved. I believe I read [in Issue 148](https://github.com/OnionUI/Onion/issues/148) that Onion doesn't support `.dat` files, so I am assuming using a `hiscore.dat` file is not the way forward. From other research it seems [it's built into the mame-2003-plus libretro core](https://docs.libretro.com/library/mame2003_plus/#high-scores). 

So, am I missing a setting or is this something that should just work? I don't have a large romset for arcade games, perhaps ten or fewer.

**Comment by schmurtzm** [comment, marked as answer]
DAT files should be supported with mame 2003 + , do you have the file  BIOS\mame2003-plus\hiscore.dat ? May be try to delete it to let the core create it again ?

Which game allows to create a highscore quickly 😅 ?

**Reply by papalozarou** [reply]
Update to 4.2.2 release and a delete just in case seemed to fix it.
